subhub/db.py

```diff
@@ -61,7 +61,7 @@ class Meta:
     def new_user(
         self, uid: str, origin_system: str, cust_id: Optional[str] = None
     ) -> SubHubAccountModel:
-        return self.model(
+        return self.model(  # type: ignore
```

Comment:
why are we ignoring the model type anyway? isn't this the kind of thing that we to check typing for?

Reply:
mypy throws a mismatch, it claims to be looking for SubHubAccountModel but returns SubHubAccountModel@427  (not sure the exact number here, going from memory), after spending more than an hour trying to determine a solution I finally went with ignoring this.

Reply:
I think because you defined the model twice no?

Reply:
```
 sidler@76  ~/repos/mozilla/subhub   master  rg SubHubAccountModel
subhub/db.py
18:    class SubHubAccountModel(Model):
29:    return SubHubAccountModel
34:class SubHubAccountModel(Model):
47:        class SubHubAccountModel(Model):
59:        self.model = SubHubAccountModel
63:    ) -> SubHubAccountModel:
71:    def get_user(self, uid: str) -> Optional[SubHubAccountModel]:
80:    def save_user(user: SubHubAccountModel) -> bool:
```
see how the class is defined twice?

Reply:
@scottidler I agree in theory but the SubHubAccountModel defined on ln 35 is done so solely for type-checking, without it SubHubAccountModel is not able to be referenced as a return value.